packages/salesforcedx-vscode-core/src/conflict/persistentStorageService.ts

```diff
@@ -70,6 +72,8 @@ export class PersistentStorageService {
   }
 
   public makeKey(type: string, fullName: string): string {
-    return `${type}#${fullName}`;
+    const orgUserName = workspaceContext.username;
```

Comment:
I didn't have error handling of 1) orgUserName undefined and 2) projectPath empty here because I believe they should trigger 1) `notificationService.showErrorMessage(nls.localize('missing_default_org'));` and 2) `workspaceChecker` in `forceSourceDiff.ts` before entering `PersistentStorageService.ts`. Let me know if I miss anything! :)